llama_index/download/download_utils.py

```diff
@@ -314,3 +322,51 @@ def download_llama_module(
     spec.loader.exec_module(module)  # type: ignore
 
     return getattr(module, module_class)
+
+
+def download_llama_dataset(
```

Comment:
This seems like a copy of `download_llama_dataset()` -- Should we control the behavior through kwargs instead?
Unsure tbh, it is getting a little crazy in here haha

Reply:
hehe ya i hear you. Let me think on this and see if we can sensibly clean it up using kwargs instead.